### PR TITLE
athanasiadis2022weaklysupervised

### PR DESCRIPTION
**Pull request by georgmosh**
## Project Name

Write the `athanasiadis2022weaklysupervised` here, and as well as rename the PR name to reflect the `<paper_citekey>` for quick reviewing. 

## Compiled PDF

Add the link to the compiled pdf here. Example : [Compiled Pdf](https://github.com/georgmosh/MLRC/blob/main/2021/athanasiadis2022weaklysupervised/journal/article.pdf)

## Metadata Checklist

Ensure that you have added the following metadata information (put an x to the checkbox items) in `<paper_citekey>/journal/metadata.yaml`.

- [x] `title` field exists and it starts with `[Re] `.
- [x] `authors` contain author information with affiliations.
- [x] [ORCID id](https://orcid.org/) is provided for authors.
- [x] One contact author is added using `*`.
- [x] Github `url` is provided in `code` section.
- [x] [SWH identifier](https://archive.softwareheritage.org/save/) is provided in `swh`.
- [x] `replication` section is populated with the necessary information about the paper you reproduced.
- [x] `abstract` field is populated with your paper abstract (including double quotes)
- [x] `domain` should contain "ML Reproducibility Challenge 2021"
- [x] `review` section contains the url to the OpenReview forum of your paper.
- [x] Fields `contributors`, `dates`, `article` and `journal` are kept blank, as those will be populated by the Area Chairs / ReScience editors.

## Build Errors

If your build had any errors locally, please add it here. You can upload your build.log in a [Pastebin](https://pastebin.com/U3pW3kSr) link and put it here.

### CODE REVIEW

File: 2021/athanasiadis2022weaklysupervised/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,39 @@
+% Import packages here
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage{hyperref}
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
```
Comment (koustuvsinha) [inline 2021/athanasiadis2022weaklysupervised/openreview/packages.tex:6]: Please remove this import (stated in README).

File: 2021/athanasiadis2022weaklysupervised/openreview/packages.tex [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage{graphicx}       % Allows including images
+\usepackage{amsmath}
+\usepackage{bm}             % bold and italics maths at the same time
+\usepackage{amssymb}        % the definition equal triangle symbol
+\usepackage{xcolor}         % colors
```
Comment (koustuvsinha) [inline 2021/athanasiadis2022weaklysupervised/openreview/packages.tex:17]: And also remove the xcolor import.

File: 2021/athanasiadis2022weaklysupervised/openreview/packages.tex [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+\usepackage{float} % fix position
+\usepackage{times}
+\usepackage{epsfig}
+\usepackage{graphicx}
+\usepackage{amsmath}
+\usepackage{amssymb}
+\usepackage{amsthm}
+\usepackage{subfig}
+\usepackage{subfigure}
+\usepackage[labelformat=simple]{subcaption}
```
Comment (koustuvsinha) [inline 2021/athanasiadis2022weaklysupervised/openreview/packages.tex:29]: If you are importing `subfigure`, then do not import `subcaption` - these two doesn't play well with each other.

**Comment by georgmosh** [comment]
Hey koustuvsinha; thank you for the hints. I made the changes indicated but the bibliography and reference issues I described above unfortunately still persist.

**Comment by koustuvsinha** [comment]
Hi @georgmosh , unfortunately that doesn't work for us as we need to rebuild every paper after acquiring publishing information. You would need to submit a version which compiles using `build_all.sh`.

I went through your submission, and it has tons of latex errors which Overleaf conveniently masks out during building. You might think this is good, but unfortunately this is a bad practice employed by Overleaf. You should check the build logs carefully and fix those issues. Typically, you should check the lines with "error" or beginning with exclamation marks, and they follow a line number to fix. There are many issues in your current draft:

- Do not use `\label` within `\begin{align*} ... \end{align*}` blocks. For some reason amsmath doesn't like it.
- Your usage of superscripts is incorrect in many places. L82, for example: `$R_j^{u}^{(n)}$` -> this should be `$R_j^{{u}^{(n)}}$` (enclosing brackets after the first superscript). 

Given this is a fairly large non-trivial amount of corrections required, I grant an extension to your submission by Friday 6th, EOD.

**Comment by georgmosh** [comment]
@koustuvsinha No problem at all. Now it should be running smoothly without issues.

**Comment by koustuvsinha** [comment]
Thanks, compilation works on my end!

**Comment by koustuvsinha** [comment]
@georgmosh You might have deleted some files from `2021` folder which is why the build is failing. Please revert those commits, thanks!

**Comment by georgmosh** [comment]
Hi @koustuvsinha I suspect some files were omitted while commiting from terminal despite running `git add --all`. I'll do a check right away but could you let me know the names of the files that are missing?

**Comment by koustuvsinha** [comment]
Hi, there are merge conflicts in this PR as you attempted to manually change the files you modified. An easy way to do it would be to checkout the offending files based on an earlier commit to main, such as this:

```
git checkout -f 53ebd92 -- 2021/accepted.bib 2021/check_yaml.py 2021/editorial/
```

Where, 53ebd92 is an earlier commit on the main branch. Then, commit your state after checking out those files from this earlier commit, and we will merge the PR.

**Comment by georgmosh** [comment]
To get things working I'll revert to the state after running setup.sh (060d8) and redo everything.

**Comment by georgmosh** [comment]
Hi @koustuvsinha [koustuvsinha](https://github.com/koustuvsinha), I have reverted all my commits and done everything from the first step with only changing the latex source files, I really hope everything is ok now and thank you for your time!

**Pull request merged**